src/librustc_trait_selection/traits/select.rs

```diff
@@ -503,6 +503,13 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
                     Err(_) => Ok(EvaluatedToErr),
                 }
             }
+
+            ty::Predicate::WellFormedConst(_constant) => {
+                // FIXME: should we call wf::const_obligations here?
+                //
+                // We do not consider const WF when evaluating trait matches.
+                Ok(EvaluatedToOk)
```

Comment:
If `WellFormed` is handled, so should `WellFormedConst`. `WellFormed` can result in `WellFormedConst` obligations, for example.